arrays/arrays.c

```diff
@@ -21,11 +21,16 @@ typedef struct Array {
  *****/
 Array *create_array (int capacity) {
   // Allocate memory for the Array struct
+  Array *arr = malloc(sizeof(Array));
 
   // Set initial values for capacity and count
+  arr->capacity = capacity;
+  arr->count = 0;
 
   // Allocate memory for elements
+  arr->elements = malloc(capacity * sizeof(char *));
```

Comment:
You may want to consider using calloc here. Mainly just because it initializes a set number of values in the array and also it sets each value to NULL which can be useful if you later on use a function that checks if an arr->elements[i] value is == NULL. I'm not sure if you get that with malloc, but you can try it if you want. You can still create individual array elements with malloc.